### PR TITLE
@responsiveImage directive in Gatsby

### PR DESCRIPTION
**Pull request by pmelab**
## Package(s) involved

`@amazeelabs/gatsby-source-silverback`
`@amazeelabs/gatsby-silverback-cloudinary`

## Description of changes

Provide a `@responsiveImage` directive to Gatsby that will handle cloudinary image processing
for the field it is attached to. Works exactly like the same directive on Drupal side.

## Motivation and context

Avoid duplicate declaration of fields that emit responsive images in configuration of `@amazeelabs/gatsby-silverback-cloudinary`.

## How has this been tested?

- [x] Manually
- [x] Unit tests
- [x] Integration tests

### CODE REVIEW

**Comment by pmelab** [comment]
Current dependencies on/for this PR:
* development
  * **PR #1418** <a href="https://app.graphite.dev/github/pr/AmazeeLabs/silverback-mono/1418" target="_blank">Graphite</a> 
    * **PR #1420** <a href="https://app.graphite.dev/github/pr/AmazeeLabs/silverback-mono/1420" target="_blank">Graphite</a> 
      * **PR #1422** <a href="https://app.graphite.dev/github/pr/AmazeeLabs/silverback-mono/1422" target="_blank">Graphite</a>  👈
        * **PR #1423** <a href="https://app.graphite.dev/github/pr/AmazeeLabs/silverback-mono/1423" target="_blank">Graphite</a> 
          * **PR #1424** <a href="https://app.graphite.dev/github/pr/AmazeeLabs/silverback-mono/1424" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/AmazeeLabs/silverback-mono/1422?utm_source=stack-comment).

File: packages/npm/@amazeelabs/gatsby-source-silverback/src/helpers/schema.test.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { directives } from '@amazeelabs/test-directives';
+import type { Gat } from 'gatsby/dist/schema/types';
```
Comment (Leksat) [inline packages/npm/@amazeelabs/gatsby-source-silverback/src/helpers/schema.test.ts]: Oh my Gat! What's that? 😄 

Do we have static tests turned off for test files on purpose? Or why static test didn't catch this?

**Review by Leksat — APPROVED** [review]

File: packages/npm/@amazeelabs/gatsby-source-silverback/src/helpers/schema.test.ts (hunk 1 above)
Comment (pmelab) [reply packages/npm/@amazeelabs/gatsby-source-silverback/src/helpers/schema.test.ts]: Indeed, tests where excluded. Fixed that.

**Comment by pmelab** [comment]
### Merge Activity

* **Sep 27, 6:18 AM**: @pmelab started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/AmazeeLabs/silverback-mono/1422).
* **Sep 27, 6:30 AM**: [Graphite](https://app.graphite.dev/github/pr/AmazeeLabs/silverback-mono/1422) couldn't merge this pull request because a downstack PR #1420 failed to merge.

**Pull request merged**